chromium/background.js

```diff
@@ -207,8 +212,21 @@ AppliedRulesets.prototype = {
     return null;
   },
 
+  addHostnameToTab: function(tabId, ruleset_name, hostname) {
```

Comment:
@jsha this is just because this was forked from the first iteration of `firefox-webextensions-core` - should I rebase it?